misk-gcp/src/test/kotlin/misk/cloud/gcp/tracing/TracingLoggingEnhancerTest.kt

```diff
@@ -50,6 +50,10 @@ class NoopWriter : Writer {
   override fun close() {
   }
 
+  override fun flush() : Boolean {
+    return true
```

Comment:
?

Reply:
This is test-only code. See commit message. They added a flush method to their Writer interface.